phase1/gce/do

```diff
@@ -50,8 +53,8 @@ fetch_kubeconfig() {
     # succeeds, then stdout is tainted by ssh initialization output, so we echo
     # a marker to indicate the start of the file, and delete all lines up to and
     # including it.
-    if gcloud compute ssh --project "$PROJECT" --zone "$ZONE" "$MASTER" --command "echo STARTFILE; sudo cat /etc/kubernetes/admin.conf" > .tmp/kubeconfig.raw; then
-      sed '0,/^STARTFILE$/d' .tmp/kubeconfig.raw > .tmp/kubeconfig.json
+    if gcloud compute ssh --project "$PROJECT" --zone "$ZONE" "$MASTER" --command "echo STARTFILE; sudo cat /etc/kubernetes/admin.conf" > ${TMP_DIR}/kubeconfig.raw; then
+      sed -e '1!b' -e '/^STARTFILE$/d' ${TMP_DIR}/kubeconfig.raw > ${CLUSTER_NAME}/kubeconfig.json
```

Comment:
I did missed this part earlier.
I tried a lot to use ssh quiet options and failed to make it work. so i resorted to sed again. This one will work for almost all environment i hope. I tested on `docker-dev` environment and also from the `gcr.io/k8s-testimages/e2e-kubeadm:v20170728-e1f1ad9b` and in both this sed command works.